.golangci.yml

```
@@ -0,0 +1,49 @@
+service:
+  golangci-lint-version: 1.21.0 # Keep this in sync with .circleci/config.yml and Jenkinsfile.pr
+
+run:
+  timeout: 5m
+  modules-download-mode: readonly
+
+linters-settings:
+  gofmt:
+    simplify: true
+  govet:
+    check-shadowing: true
+    enable-all: true
+
```

I like the option for `goimports`, that forces you to split the imports into three parts. This would also be consistent with the Mattermost server.

```suggestion
  goimports:
    local-prefixes: github.com/mattermost/mattermost-marketplace

```